src/cli/db.ts

```diff
@@ -0,0 +1,66 @@
+import { flags } from '@oclif/command'
+import { OutputFlags } from '@oclif/parser'
+
+import { loggingFactory } from '../logger'
+import BaseCommand from '../utils'
+import DbMigration from '../../migrations'
+
+const logger = loggingFactory('cli:db')
+
+export default class DbCommand extends BaseCommand {
+  static hidden: boolean;
+  static flags = {
+    ...BaseCommand.flags,
+    up: flags.boolean({
+      char: 'm',
+      description: 'Migrate DB'
+    }),
+    down: flags.boolean({
+      char: 'u',
+      description: 'Undo db migration'
+    }),
+    migration: flags.string({
+      char: 'm',
+      description: 'Migration file',
+      multiple: true
+    })
+  }
+
+  static description = 'DB migration'
+
+  static examples = [
+    '$ rif-pinning db --up',
+    '$ rif-pinning db --down',
+    '$ rif-pinning db --up --from 01-test --to -0-test',
+    '$ rif-pinning db --up --migration 01-test --migration 02-test',
+    '$ rif-pinning db --up --db ./test.sqlite --from 01-test --to 09-test',
+    '$ rif-pinning db --down --db ./test.sqlite --from 01-test --to 09-test',
+    '$ rif-pinning db --up --db ./test.sqlite --from 01-test --to 09-test'
```

Comment:
```suggestion
    '$ rif-pinning db --up --to 0-test',
    '$ rif-pinning db --up --migration 01-test --migration 02-test',
    '$ rif-pinning db --up --db ./test.sqlite --to 09-test',
    '$ rif-pinning db --down --db ./test.sqlite --to 09-test'
```

Reply:
Should i revert “to” flag? I already remove both

Reply:
Oh I haven't noticed 😅 Hmm I think it would be good to have `--to` because if we migrate down it makes sense to say where to... But if it is left out I would default to latest migration...

Reply:
Done

Reply:
I will also add another flag `create` which is generate migration file for us